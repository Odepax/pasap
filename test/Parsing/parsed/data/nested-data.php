<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<group-set <?= \Pasap\Pasap::data([ 'two' => 2, 'three' => 3 ]) ?>>
			<list-set <?= \Pasap\Pasap::data([ 'one' => 1, 'two' => 22 ]) ?> />
		</group-set>

		<group-set <?= \Pasap\Pasap::data([ 'two' => 2, 'three' => 3 ]) ?>>
			<list-scope <?= \Pasap\Pasap::scope([ 'one' => 1, 'two' => 22 ]) ?> />
		</group-set>

		<group-scope <?= \Pasap\Pasap::scope([ 'two' => 2, 'three' => 3 ]) ?>>
			<list-set <?= \Pasap\Pasap::data([ 'one' => 1, 'two' => 22 ]) ?> />
		</group-scope>

		<group-scope <?= \Pasap\Pasap::scope([ 'two' => 2, 'three' => 3]) ?>>
			<list-scope <?= \Pasap\Pasap::scope([ 'one' => 1, 'two' => 22 ]) ?> />
		</group-scope>
	</body>
</html>
