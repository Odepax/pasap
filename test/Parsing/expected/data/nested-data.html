<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Document</title>
	</head>
	<body>
		<section class="DataSet">
			<dl>
				<dt>One</dt> -- <dd></dd>
				<dt>Two</dt> -- <dd>2</dd>
				<dt>Three</dt> -- <dd>3</dd>
			</dl>
			<div class="children">
				<dl class="DataSet">
					<dt>One</dt> -- <dd>1</dd>
					<dt>Two</dt> -- <dd>22</dd>
					<dt>Three</dt> -- <dd></dd>
				</dl>
			</div>
		</section>

		<section class="DataSet">
			<dl>
				<dt>One</dt> -- <dd></dd>
				<dt>Two</dt> -- <dd>2</dd>
				<dt>Three</dt> -- <dd>3</dd>
			</dl>
			<div class="children">
				<dl class="DataScope">
					<dt>One</dt> -- <dd>1</dd>
					<dt>Two</dt> -- <dd>22</dd>
					<dt>Three</dt> -- <dd></dd>
				</dl>
			</div>
		</section>

		<section class="DataScope">
			<dl>
				<dt>One</dt> -- <dd></dd>
				<dt>Two</dt> -- <dd>2</dd>
				<dt>Three</dt> -- <dd>3</dd>
			</dl>
			<div class="children">
				<dl class="DataSet">
					<dt>One</dt> -- <dd>1</dd>
					<dt>Two</dt> -- <dd>22</dd>
					<dt>Three</dt> -- <dd></dd>
				</dl>
			</div>
		</section>

		<section class="DataScope">
			<dl>
				<dt>One</dt> -- <dd></dd>
				<dt>Two</dt> -- <dd>2</dd>
				<dt>Three</dt> -- <dd>3</dd>
			</dl>
			<div class="children">
				<dl class="DataScope">
					<dt>One</dt> -- <dd>1</dd>
					<dt>Two</dt> -- <dd>22</dd>
					<dt>Three</dt> -- <dd>3</dd>
				</dl>
			</div>
		</section>
	</body>
</html>
